frontend/src/styles/App.css

```diff
@@ -44,6 +44,11 @@ span {
   color: inherit !important;
 }
 
+/* Logo styling */
+img {
+  padding-bottom: 1rem;
+}
+
```

Comment:
For consistency, let's keep these implemented using Bootstrap utilities. This css change can be removed and rather have this in the components:
```js
<img src={logo} alt="BudgetCtrl logo" className="pb-3"/>
```

Reply:
Suggested change ready